.github/workflows/build-code.yml

```diff
@@ -24,16 +24,16 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node_version }}
-          cache: 'yarn'
+          cache: 'npm'
 
       - name: Install dependencies
-        run: yarn install --frozen-lockfile
+        run: npm ci
```

Comment:
Do we need to use `npm ci --legacy-peer-deps` here that the build passes?

Reply:
That would work.